documentation/docs/07-misc/02-testing.md

```diff
@@ -246,6 +248,82 @@ test('Component', async () => {
 
 When writing component tests that involve two-way bindings, context or snippet props, it's best to create a wrapper component for your specific test and interact with that. `@testing-library/svelte` contains some [examples](https://testing-library.com/docs/svelte-testing-library/example).
 
+_With Storybook_
+
+[Storybook](https://storybook.js.org?ref=svelte-docs) is the industry standard for developing and documenting UI components, and it can also be used to test your components. A storybook is made up of [stories](https://storybook.js.org/docs/writing-stories?ref=svelte-docs&renderer=svelte)—isolated examples of your components in different states and with simulated interactions. These stories can also function as [component tests](https://storybook.js.org/docs/writing-tests?ref=svelte-docs&renderer=svelte), with no additional code needed. They're run with Vitest's browser mode, which renders your components in a real browser for the most realistic testing environment.
+
+To get started, first install Storybook ([using Svelte's CLI](/docs/cli/storybook)) in your project if you haven't already:
+
+```sh
+npx sv add storybook
+```
+
+When prompted, choose the recommended configuration that includes testing features.
+
+If you already have Storybook set up, but are not yet using the testing features, first upgrade to the latest version:
+
+```sh
+npx storybook@latest upgrade
+```
+
+Then, install and configure Storybook's Vitest addon:
+
+```sh
+npx storybook add @storybook/addon-vitest
+```
+
+That `add` command will install and register the Vitest addon. It will also inspect your project's Vite and Vitest setup, and install and configure them with sensible defaults, if necessary. You may need to adjust the [configuration](https://storybook.js.org/docs/writing-tests/integrations/vitest-addon?ref=svelte-docs&renderer=svelte#options) to fit your project's needs.
```

Comment:
This mostly seems self-explanatory and a bit verbose. Could probably squeeze it in as just a few words when introducing the command above:
```suggestion
```